### PR TITLE
Leinard/contacts

### PR DESCRIPTION
**Pull request by leinardjg**

### CODE REVIEW

File: src/main/java/Contacts/Contact.java [hunk 1]
```diff
@@ -0,0 +1,256 @@
+package Contacts;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+public class Contact extends UI {
+
+  // variables
+
+  private Scanner scanner = new Scanner(System.in);
```
Comment (NOlson95) [inline src/main/java/Contacts/Contact.java]: instantiate scanner with InputLibrary.getScanner() instead.

File: src/main/java/Contacts/Contact.java [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+      input = input.trim();
+      if (input.equals("")) {
+        System.out.println("Name can't be empty!");
+      } else {
+        this.setName(input);
+      }
+    }
+  }
+
+  void promptEmail() {
```
Comment (NOlson95) [inline src/main/java/Contacts/Contact.java]: Could add data validation in the future to ensure correct format of data being entered.

File: src/main/java/Contacts/Contact.java [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+  }
+
+  void promptPhoneNo() {
+
+    String input;
+
+    System.out.println(
+        "Enter phone number(s) (leave empty for none; separate multiple with commas): ");
+    input = scanner.next();
+    input = input.replaceAll("\\s", "");
```
Comment (rynoV) [inline src/main/java/Contacts/Contact.java]: Regular expressions are kind of hard to read, could you put a comment explaining them when you use them please?

File: src/main/java/Contacts/Contact.java [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+    System.out.println("Enter new name: ");
+    input = scanner.next().trim();
+    if (input.equals("")) {
+      System.out.println("Name can't be empty!");
+    } else {
+      this.setName(input);
+    }
+  }
+
+  void editEmail() {
```
Comment (rynoV) [inline src/main/java/Contacts/Contact.java]: We should probably allow for editing individual emails/phone numbers in the future

**Pull request merged**